### ISSUE TITLE
question about dataset for only train background without foreground？

### ISSUE THREAD

**Issue by flyinghippof** [OPEN-AIR-SUN/mars#97]
I want to train a scenario where there is no foreground. But I found that when loading the dataset, frames without objects will be filtered out, and simply retaining all images will result in corresponding other errors. How should I modify this situation?

**Comment by xBeho1der** [comment]
Hi! Thanks for your interest! Could you please provide more information about the issue you are encountering?

**Comment by flyinghippof** [comment]
Thank you very much for your prompt response！
I found that the division of training and testing sets is based on ”visible_objects“。 If there are no ”visible_objects“ in my scene, some errors will occur due to "indices" being zero or None。
For example：
”obj_info = torch.cat([obj_nodes_tensor[i : i + 1] for i in indices], dim=0)“

![image](https://github.com/OPEN-AIR-SUN/mars/assets/72117809/41d55819-f37a-4fe5-b763-52fb9a8c2069)

**Comment by wuzirui** [comment]
Hi! If you only need the background node, I would suggest using the original Nerfacto model from nerfstudio. Because the main goal of our repository is to compose nodes together rather than obtaining a single node.
